### ISSUE TITLE
Invoices include 'how to pay' information - including Stripe link

### ISSUE THREAD

**Issue by kirstenalarsen** [openfoodfoundation/wishlist#22]
## Description

After openfoodfoundation/openfoodnetwork#5208 invoices sent after changes to orders will have the correct information about balances due.

A standard feature on a normal invoice is to include information on how the Customer can pay that invoice. This should be relatively easy to add to our invoices, so that Hubs can just send them out and Customers can pay any balance owing

Shops could add payment information to be printed on their invoices in the Business Details settings page

![image.png](https://images.zenhubusercontent.com/54b83fb37c6ccc6277a6ea46/d3fb71a6-1ad8-4304-9f83-c4afc560590c)

[on hold as I pursue whether it is actually possible to just put a custom stripe link in there .. ]
[of course would be even better if it could open a payment session to take the payment and reconcile it against the order, but i reckon that's getting well out of papercut land - @luisramos0 stripe guru open to your thoughts] 

## Acceptance Criteria & Tests

1. Hub / Shop can send an updated invoice with correct balance (#5208)
2. Customer can easily pay their balance due 

NB. This proposed solution still leaves the payment unreconciled i.e. not matched against the amount owing in OFN - that will need to be done manually.

**Comment by andrewpbrett** [comment]
Agree with the consensus in the comments so far here... adding a text field is papercut sized. The mechanism for letting a customer pay for a changed invoice with a balance due by credit card seems really important (!) but definitely outside papercut territory.

**Comment by jaycmb** [comment]
Would just adding the field and the hub managers having to create a new link for each invoice via stripe still be wanted?

**Comment by jaycmb** [comment]
Agreed that this is a great feature request but needs some more inception.
